src/main/java/gregtech/common/metatileentities/multi/multiblockpart/MetaTileEntityItemBus.java

```diff
@@ -41,14 +48,19 @@ public MetaTileEntity createMetaTileEntity(MetaTileEntityHolder holder) {
 
     @Override
     public void update() {
+        IItemHandlerModifiable inventory = (isExportHatch ? this.getExportItems() : this.getImportItems());
         super.update();
         if (!getWorld().isRemote && getOffsetTimer() % 5 == 0) {
             if (isExportHatch) {
                 pushItemsIntoNearbyHandlers(getFrontFacing());
             } else {
                 pullItemsFromNearbyHandlers(getFrontFacing());
             }
+            if (isAutoCollapse()){
```

Comment:
You could change `inventory` to be `NotifiableItemStackHandler` and then add a check here: `&& notifiedItemInputList.contains(inventory)` (Or the same for output)